docs/source/user-guide/nlp/quickstarts/plot_getting_started.py

```diff
@@ -0,0 +1,210 @@
+   :ref:`Token Classification <nlp_supported_token_classification>` label.
+3. Your models predictions (see :doc:`Supported Tasks </user-guide/nlp/supported_tasks>` for info on supported formats).
+
+If you don't have deepchecks installed yet:
+
+.. code:: python
+
+    import sys
+    !{sys.executable} -m pip install deepchecks[nlp] -U --quiet #--user
+
+Some properties calculated by deepchecks.nlp require additional packages to be installed. You can
+install them by running:
+
+.. code:: python
+
+    import sys
+    !{sys.executable} -m pip install langdetect>=1.0.9 textblob>=0.17.1 -U --quiet #--user
+
+Finally, we'll be using the CatBoost model in this guide, so we'll also need to install it:
+
+.. code:: python
+    import sys
+    !{sys.executable} -m pip install catboost -U --quiet #--user
+
+"""
+
+#%%
+# Load Data & Create TextData Objects
+# ===================================
+# For the purpose of this guide we'll use a small subset of the
+# `tweet emotion <https://github.com/cardiffnlp/tweeteval>`__ dataset:
+
+# Imports
+from deepchecks.nlp import TextData
+from deepchecks.nlp.datasets.classification import tweet_emotion
+
+# Load Data
+train, test = tweet_emotion.load_data(data_format='DataFrame')
+train.head()
+
+#%%
+#
+# We can see that we have the tweet text itself, the label (the emotion) and then some additional metadata columns.
+#
+# We can now create a :class:`TextData <deepchecks.nlp.TextData>` object for the train and test dataframes.
+# This object is used to pass your data to the deepchecks checks.
+#
+# To create a TextData object, the only required argument is the text itself, but passing only the text
+# will prevent multiple checks from running. In this example we'll pass the label as well and also provide
+# metadata (the other columns in the dataframe) which we'll use later on in the guide. Finally, we'll also
+# explicitly set the index.
+#
+# .. note::
+#    The label column is optional, but if provided you must also pass the task_type argument, so that deepchecks
+#    will know how to interpret the label column.
+#
+
+train = TextData(train.text, label=train['label'], task_type='text_classification',
+                 index=train.index, metadata=train.drop(columns=['label', 'text']))
+test = TextData(test.text, label=test['label'], task_type='text_classification',
+                index=test.index, metadata=test.drop(columns=['label', 'text']))
+
+#%%
+# Building a Model
+# ================
+#
+# In this example we'll train a very basic model for simplicity, using a CatBoostClassifier trained over the
+# embeddings of the tweets. In this case these embeddings were created using the OpenAI GPT-3 model.
+# If you want to reproduce this kind of basic model for your own task, you can calculate your own embeddings, or use
+# our :func:`calculate_embeddings_for_text <deepchecks.nlp.utils.calculate_embeddings_for_text>`
```

Comment:
This is a very niche thing (I expect very few users to use this rather than an actual text model e.g. bert) so I am very reluctant to waste any more words on it.